pyproject.toml

```diff
@@ -44,7 +44,9 @@ bandit = "*"
 pylint = "*"
 pylint-django = "*"
 pydocstyle = "*"
-flake8 = "*"
+# we need to pin flake8 because of package dependencies that cause it to downgrade and 
+# therefore cause issues with linting since older versions do not take .flake8 as config
+flake8 = "^3.9.2"
```

Comment:
This is a different approach than we've taken in other repos in this organization, where the fix there was to explicitly invoke flake8 as `flake8 --config .flake8`, e.g. https://github.com/nautobot/nautobot-plugin-netbox-importer/pull/75/files - is that alternative preferable?

Also, what dependencies do we have that were causing a downgrade to older versions of flake8?